test-two/src/index.js

```diff
@@ -3,24 +3,49 @@ import { getUsersForOrganisation } from "./services/github";
 
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = "cf33f5689e7e8399aadb5cbaae8ef51bc2be7c66"; // add your GitHub API OAuth key here to increase request limit
+
+/*
+  I was unable to get my program to work in the desired way but I managed a very
+  low quality version.
+
+  My initial plan was to iterate through the program, everytime the response headers
+  had a link with 'rel=next' I would loop.
+  Or I would take the page number from 'rel=last' and loop that many times.
+  But I was unable to access the data in the header as I think it is a nested
+  java object and it kept returning "undefined".
```

Comment:
It is tricky isn't it. P.S. java and javascript are very different things.

Reply:
Yeah it was probably the hardest part of the entire assignment. I spent a long time looking at documentation and trying to convert the headers information into an object or a string so I could manipulate it but I was stumped.